core/src/main/kotlin/br/com/concrete/yosef/api/property/spacing/MarginPropertyCommand.kt

```diff
@@ -0,0 +1,49 @@
+package br.com.concrete.yosef.api.property.spacing
+
+import android.view.View
+import android.widget.FrameLayout
+import android.widget.LinearLayout
+import br.com.concrete.yosef.api.property.DynamicPropertyCommand
+import br.com.concrete.yosef.dp
+import br.com.concrete.yosef.entity.DynamicProperty
+
+/**
+ * Command class that implements the [DynamicPropertyCommand] applying
+ * padding to a view view
+ *
+ * @see [View.setPadding]
```

Comment:
Here the annotation should be directing to another View method

Reply:
done